runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -1271,6 +1273,66 @@ public void testKeyCommitTooLargeException() throws Exception {
     assertTrue(foundErrors);
   }
 
+  @Test
+  public void testOutputKeyTooLargeException() throws Exception {
```

Comment:
either add test or add to these tests that values lower than the limit go through successfully

Reply:
This is tested by the fact that after the first exception is thrown, the DoFn outputs smaller values that we are checking against in the output. I'll also augment the test to pass values that never throw exception as well though.